main.tex

```diff
@@ -347,14 +347,45 @@ \subsection{Support of Astropy}
 Different funding models have been proposed for support of \astropy
 (e.g., \citealt{2016arXiv161003159M}), but a long-term plan
 for sustainability has not yet been established.
-The \astropy project has the ability to accept financial contributions
+The \astropy Project has the ability to accept financial contributions
 from institutions or individuals through the NumFOCUS\footnote{NumFOCUS
 is a 501(c)(3) nonprofit that supports and promotes world-class, innovative,
-open source scientific computing.} organization. NumFOCUS has, to date, covered the direct costs incurred by the \astropy project.
+open source scientific computing.} organization. NumFOCUS has, to date, covered the direct costs incurred by the \astropy Project.
+
+\subsection{Reuse of the scientific Python ecosystem}
+
+The \astropy Project is built on a philosophy of building from the existing
+Python scientific ecosystem wherever possible. Many of the enabling technologies
+like \texttt{numpy}, \texttt{scipy}, \texttt{matplotlib}, or \texttt{cython},
+are necessary as the core underpinnings of \astropy packages.  Often this means
+simply means using these packages as ``building blocks'' (E.g., the ubiquitous
+use of \texttt{numpy} arrays throughout \astropypkg).  In other cases this means
+wrappers around more general algorithms that make them more convenient for
+astronomy use cases (E.g., the model-fitting in the \package{astropy.modeling}
```

Comment:
you consistently use `e.g.` capitalized. I wonder whether that indeed the right format? If yes, we need to change all the other lowercase occurrences elsewhere in the paper.

Reply:
It's not a start of sentence so should be lower case. I tend to shy away from using `e.g.` and prefer to use words such as "such as" or "for example" but I admit I don't know the AAS style preference.

Reply:
Good points both.  I'm going to take the middle ground and follow @timj's rule of e.g. when not at the start of the sentence, but we'll leave it for the copyeditor to tell us if "for example" is better (since e.g./i.e. is my *personal* default style...)